common/views/components/ResponsiveImage/ResponsiveImage.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+
+import { useRef, useEffect } from 'react';
+
+import { convertImageUri } from '../../../utils/convert-image-uri';
+import { classNames } from '../../../utils/classnames';
+import { imageSizes } from '../../../utils/image-sizes';
+
+type Props = {
+  url: string,
+  width: number,
+  height: ?number,
+  sizesQueries: ?string,
```

Comment:
Why is this optional? Isn't this the point of a `ResponsiveImage`?

Reply:
sizes is optional, if you don't have it the browser assumes the image is always displayed at 100% width, but will still choose the appropriate image from the src-set, based on the browser size

Reply:
👍 just so long as we don't start having `sizesQuery={null}` all over the shop.

Reply:
would `sizesQueries?: string` be better?

Reply:
> would `sizesQueries?: string` be better?

Feels like not, as then you could accidentally omit it rather than explicitly saying `sizesQuery={null}`